ui/app/components/app/index.scss

```diff
@@ -1,3 +1,5 @@
+@import url('fonts/muli.css');
+@import url('fonts/poppins.css');
```

Comment:
mostly just curious -- what's the purpose of these new `fonts` CSS imports?

Reply:
Muli and Poppins are Brave’s main typography for headers and copy text, so we have these for consistency. The previous style job imported them